cfgov/jinja2/v1/_includes/molecules/image-text-25-75.html

```diff
@@ -10,25 +10,32 @@
    Create an image and text 25/75 molecule.
    See [GHE]/flapjack/Modules-V1/wiki/25-75-Image-&-Text
 
-   value:            An object with the following options for value.
+   value:              Object defined from a StreamField block.
 
-   value.image:      A image object containing the URL & alt text for screenreaders
-                        of the image to be placed adjacent to the text.
+   value.image:        A image object containing the image to be
+                       placed adjacent to the text.
 
-   value.heading:    Heading of the text content.
-   value.body:       Body of the text content.
+   value.image.upload: An image object.
 
-   value.links:      A list of links containing URL & Text of the link/button.
+   value.image.alt:    String for alt text for screenreaders.
 
-   value.has_rule:   Whether there is a top gray rule.
-                        Default is false.
+   value.heading:      Heading of the text content.
+
+   value.body:         Body of the text content.
+
+   value.links:            A tuple of items to create a list of links, containing:
+
+   value.links[i].url:     A string for the URL of the link.
+
+   value.links[i].text:    A string for the text of the link.
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2032/commits/e95d3aa47240421b21ff48e05041ebf63b1cf6b5